CONTRIBUTE.adoc

```diff
@@ -29,6 +29,20 @@ Don’t get discouraged if you don't get an immediate response, this is a side-p
 
 Testing existing features is a good way to contribute to the project, too!
 
+== Modifying The Grammar
```

Comment:
I understand that it can be a bit of a pain for some users to install `make`, but it would be fair to provide these instructions as an _alternative_ to the `make generate`, `make build` and `make install`, not as the preferred way.

Reply:
Please review the updated PR.  I really think make should *not* be the preferred way.